packages/circuit-ui/components/SelectorGroup/SelectorGroup.spec.tsx

```diff
@@ -44,8 +44,8 @@ describe('SelectorGroup', () => {
    * Style tests.
    */
   it('should render with default styles', () => {
-    const actual = render(<SelectorGroup {...defaultProps} />);
-    expect(actual).toMatchSnapshot();
+    const { container } = render(<SelectorGroup {...defaultProps} />);
```

Comment:
This is important for snapshots because it avoids snapshotting a bunch of RTL internals on the return value of render()